src/main/java/Piece.java

```diff
@@ -27,7 +27,7 @@ public void setType(String type) {
         this.type = type;
     }
 
-    public Char getType() {
+    public String getType() {
         //Returns P for pawn, B for bishop, N for knight, 
```

Comment:
Not true. Type strings are "pawn", "knight", etc. I agree that the documentation in Piece could be clearer on this.